src/Shared/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Responses/StandardApiResponseBaseTests/WhenGettingIsActive.cs

```diff
@@ -84,13 +85,13 @@ public void And_EffectiveFrom_Today_And_No_LastDateStart_BeforeToday_Then_False(
             standard.IsActive.Should().BeFalse();
         }
 
-        [Test, AutoData]
-        public void And_EffectiveFrom_Before_Today_And_EffectiveTo_After_Today_Then_True(TestStandardResponse standard)
-        {
-            standard.StandardDates.EffectiveFrom = DateTime.UtcNow.AddDays(-1);
-            standard.StandardDates.EffectiveTo = DateTime.UtcNow.AddDays(1);
+        //[Test, AutoData]
+        //public void And_EffectiveFrom_Before_Today_And_EffectiveTo_After_Today_Then_True(TestStandardResponse standard)
```

Comment:
?

Reply:
Resolved the changes -https://github.com/SkillsFundingAgency/das-apim-endpoints/pull/834